nature_of_code/chp_03_oscillation/3_01_angular_motion.rs

```diff
@@ -11,49 +12,54 @@ fn main() {
 
 struct Model {
     angle: f32,
-    a_velocity: f32,
-    a_acceleration: f32,
+    angle_velocity: f32,
+    angle_acceleration: f32,
 }
 
 fn model(app: &App) -> Model {
     let angle = 0.0;
-    let a_velocity = 0.0;
-    let a_acceleration = 0.0001;
+    let angle_velocity = 0.0;
+    let angle_acceleration = -0.0001;
     app.new_window().size(800, 200).view(view).build().unwrap();
     Model {
         angle,
-        a_velocity,
-        a_acceleration,
+        angle_velocity,
+        angle_acceleration,
     }
 }
 
 fn update(_app: &App, model: &mut Model, _update: Update) {
-    model.angle += model.a_velocity;
-    model.a_velocity += model.a_acceleration;
+    model.angle += model.angle_velocity;
+    model.angle_velocity += model.angle_acceleration;
 }
 
 fn view(app: &App, model: &Model, frame: Frame) {
     // Begin drawing
     let draw = app.draw();
-    draw.background().color(WHITE);
+    draw.rect()
```

Comment:
This change seems to make the background a dark blue rather than white. 

![Screen Shot 2020-08-31 at 10 50 17 PM](https://user-images.githubusercontent.com/2945386/91789597-f856ec00-ebdc-11ea-99c7-5b30ca12625f.png)

Reply:
It's weird, mine it's white, but I will change it back to `WHITE`.